go-controller/pkg/util/pod_annotation.go

```diff
@@ -51,6 +53,10 @@ const (
 	OvnPodAnnotationName = "k8s.ovn.org/pod-networks"
 	// DefNetworkAnnotation is the pod annotation for the cluster-wide default network
 	DefNetworkAnnotation = "v1.multus-cni.io/default-network"
+	// SkipIPOnNetworksAnnotation specifies the NADs that don't require IP allocation
+	SkipIPOnNetworksAnnotation = "k8s.ovn.org/skip-ip-on-networks"
```

Comment:
IMHO for security reasons we should stop using annotations on pods as ways to achieve things